### ISSUE TITLE
Add support for case class generation from RAML referenced JSON schemas

### ISSUE THREAD

**Issue by sslavic** [eigengo/sbt-raml-plugin#4]
Please consider adding support for code generation from RAML, e.g. generate case classes from JSON schemas. When working with schemas, watch out, there is a bug in raml java parser: https://github.com/raml-org/raml-java-parser/issues/39

Maybe in the future some route related code can be generated as well, e.g. for spray or finatra, but data model is basis needed in any case, even when implementing routing code by hand.

**Comment by janm399** [comment]
Good idea! I would like to start with SoapUI test driver so that I can use this plugin to implement black-box integration testing.

But your idea is interesting, and potentially very useful, too!
